camera_topic: "d435/color/image_raw"
velodyne_topic: "velodyne_points"
inference_topic: "d435/color/inference_image"

mean: [0.485, 0.456 , 0.406]
std: [0.225, 0.225, 0.225]
# enable gpu-inference
use_cuda: False

camera_frame: "d435_color_optical_frame"
costmap_size: 20.0
costmap_resolution: 0.05
# change the below paths as required

serialized_engine: "/home/yeshas/PointPainting/point_painting/src/point_painting/assets/fcn32_torchscript.pt"
json_path: "/home/yeshas/PointPainting/point_painting/src/point_painting/config/color_mapping.json"